src/async_client.rs

```diff
@@ -333,6 +343,189 @@ impl AsyncClient {
+    ///
+    /// Returns a `leave_room::Response`, an empty response.
+    ///
+    /// # Arguments
+    ///
+    /// * room_id - A valid `RoomId`.
+    ///
+    pub async fn leave_room(&mut self, room_id: &RoomId) -> Result<leave_room::Response> {
+        let request = leave_room::Request {
+            room_id: room_id.clone(),
+        };
+        self.send(request).await
+    }
+
+    /// Invite the specified user by `UserId` to the given room.
+    ///
+    /// Returns a `invite_user::Response`, an empty response.
+    ///
+    /// # Arguments
+    ///
+    /// * room_id - A valid `RoomId`.
+    ///
+    /// * user_id - A valid `UserId`.
+    ///
+    pub async fn invite_user_by_id(
+        &mut self,
+        room_id: &RoomId,
+        user_id: &UserId,
+    ) -> Result<invite_user::Response> {
+        let request = invite_user::Request {
+            room_id: room_id.clone(),
+            recipient: InvitationRecipient::UserId {
+                user_id: user_id.clone(),
+            },
+        };
+        self.send(request).await
+    }
+
+    /// Invite the specified user by third party id to the given room.
+    ///
+    /// Returns a `invite_user::Response`, an empty response.
+    ///
+    /// # Arguments
+    ///
+    /// * room_id - A valid `RoomId`.
+    ///
+    /// * invite_id - A valid `UserId`.
+    ///
+    pub async fn invite_user_by_3pid(
+        &mut self,
+        room_id: &RoomId,
+        invite_id: &Invite3pid,
+    ) -> Result<invite_user::Response> {
+        let request = invite_user::Request {
+            room_id: room_id.clone(),
+            recipient: InvitationRecipient::ThirdPartyId(invite_id.clone()),
+        };
+        self.send(request).await
+    }
+
+    /// A builder to create a room and send the request.
+    ///
+    /// Returns a `create_room::Response`, an empty response.
+    ///
+    /// # Arguments
+    ///
+    /// * room - the easiest way to create this request is using the `RoomBuilder`.
+    ///
+    /// # Examples
+    /// ```ignore
+    /// use matrix_sdk::{AsyncClient, RoomBuilder};
+    ///
+    /// let mut bldr = RoomBuilder::default();
+    /// bldr.creation_content(false)
+    ///     .initial_state(vec![])
+    ///     .visibility(Visibility::Public)
+    ///     .name("name")
+    ///     .room_version("v1.0");
+    ///
+    /// let mut cli = AsyncClient::new(homeserver, Some(session)).unwrap();
+    /// assert!(cli.create_room(bldr).await.is_ok());
+    /// ```
+    ///
+    pub async fn create_room<R: Into<create_room::Request>>(
+        &mut self,
+        room: R,
+    ) -> Result<create_room::Response> {
+        let request = room.into();
+        self.send(request).await
+    }
+
+    /// Invite the specified user by third party id to the given room.
+    ///
+    /// Returns a `invite_user::Response`, an empty response.
+    ///
+    /// # Arguments
+    ///
+    /// * request - The easiest way to create a `Request` is using the `GetMessageBuilder`
```

Comment:
Can we get an example here?

I know there is one for the `GetMessageBuilder` struct, but people will need to switch to another page to find a example in that case.